src/Reflex/Dom/DHTMLX/Date.hs

```diff
@@ -44,12 +44,12 @@ createDhtmlxDateWidget elmt wstart =
     js_createDhtmlxDateWidget (pToJSVal elmt) (weekDayToInt wstart)
 
 foreign import javascript unsafe
-  "(function(){\
+  "(function(dhtmlXCalendarObject){\
     var cal = new dhtmlXCalendarObject($1);\
-    cal.setWeekStartDay($2);\
-    cal.hideTime();\
+    cal['setWeekStartDay']($2);\
```

Comment:
Wow, I thought I went through and did all this.  Oh well, I guess I was mistaken.